apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtakBehandlingService.kt

```diff
@@ -294,9 +295,17 @@ class VedtakBehandlingService(
                 behandlingId = behandlingId,
             )
 
-        val isEtterbetaling = erVedtakMedEtterbetaling(tilSamordningVedtakLocal, repository)
+        if (tilSamordningVedtakLocal.isRegulering()) {
+            logger.info("Oppretter ikke samordning ved regulering [behandlingId=$behandlingId]")
 
-        if (!samKlient.samordneVedtak(tilSamordningVedtakLocal, isEtterbetaling, brukerTokenInfo)) {
+            val vedtakEtterSvar = samordnetVedtak(behandlingId, brukerTokenInfo, tilSamordningVedtakLocal)!!
+            return VedtakOgRapid(vedtakEtterSvar.vedtak, tilSamordning, vedtakEtterSvar.rapidInfo1)
```

Comment:
`rapidInfo1` - hva er det?

Reply:
tja, er vel Mads man må spørre om bakgrunnen her, men `VedtakOgRapid` kan jo inneholde både rapid1 og rapid2, hvor sistnevnte ikke er påkrevet. Ser for meg ut som rapid2 er knyttet til noe migreringsting, som jeg jo trodde var fjernet i sin helhet fra koden nå? Så kunne man ha døpt om `rapidInfo1 -> rapidInfo`.

Reply:
```
data class RapidInfo(
    val vedtakhendelse: VedtakKafkaHendelseHendelseType,
    val vedtak: VedtakDto,
    val tekniskTid: Tidspunkt,
    val behandlingId: UUID,
    val extraParams: Map<String, Any> = emptyMap(),
)
```